test/e2e/common/util.go

```diff
@@ -433,9 +433,8 @@ func CreateFioPodDef(podName string, volName string) *corev1.Pod {
 			Containers: []corev1.Container{
 				{
 					Name:    podName,
-					Image:   "nixery.dev/shell/fio/tini",
-					Command: []string{"tini", "--"},
-					Args:    []string{"sleep", "1000000"},
+					Image:   "alpine",
```

Comment:
This looks horrible. Code changes required to make changes to containers? Why do we not read this from YAML?

Reply:
Barking at the wrong tree here :-) I would tend to agree. What changes would you imagine? Also please understand scope of this (fixing currently existing problem) and if you want some refactoring (which I would definitely recommend and it already started on some places) please create a ticket with requirements. Not sure, if I understand correctly, but I cannot see much of a difference, if the configuration YAML would be hosted in the repository as well.

Reply:
So you change fio.yaml in the repo, but you don't use that file in the test. That's basically what I was alluding too. So to me the change to fio.yaml not needed at all?

Reply:
I understood `fio.yaml` to be a file for the random user to test deploy fio when playing with mayastor. That what I've used when installing mayastor the first few times. It doesn't have connection to the tests but to the fact that when a random user from the internet wants to test it. She has a high chance that it will fails because of nixery and I regard it as a quality issue with the SW. Anyway YMMW.